config/p10k-classic.zsh

```diff
@@ -490,6 +491,12 @@ fi
   # Custom icon.
   # typeset -g POWERLEVEL9K_NODE_VERSION_VISUAL_IDENTIFIER_EXPANSION='⭐'
 
+  ################################[ go_version: golang version ]################################
+  # Golang version color.
+  typeset -g POWERLEVEL9K_NODE_VERSION_FOREGROUND=14
```

Comment:
14 is too unpredictable as it gets overridden by color pallets. Let's go with 37 instead. It's similar in shade and usually a bit less bright (go version shouldn't grab too much attention).

P.S.

Also s/NODE/GO/ here and in another file.